emsdk_manifest.json

```diff
@@ -350,6 +350,16 @@
     "activated_cfg": "NODE_JS='%installation_dir%/bin/node%.exe%'",
     "activated_env": "EMSDK_NODE=%installation_dir%/bin/node%.exe%"
   },
+  {
+    "id": "node",
+    "version": "18.13.0",
+    "arch": "aarch64",
+    "bitness": 64,
+    "windows_url": "https://github.com/Windows-on-ARM-Experiments/emsdk/releases/download/v0.1.0-alpha/node-v18.13.0-win-arm64.zip",
+    "activated_path": "%installation_dir%",
+    "activated_cfg": "NODE_JS='%installation_dir%/node%.exe%'",
```

Comment:
We should keep the package structure consistend with the upstream one. Let me check how the upstream package looks like or whether the bin shall not be added during unpacking...

Reply:
I will use '%installation_dir%/bin